index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="description" content="Youtube is cloned as a project of Microverse Curriculum">
+        <meta charset="utf-8">
+        <title>Youtube Clone</title>
+        <link rel="stylesheet" href="CSS/reset.css">    
+        <link rel="stylesheet" href="CSS/styles.css"> 
+    
+    </head>
+
+    <body>
+    
+        <nav id="navbar">
+            
+            <div id="menu" >
+                <img src="images/menuButton.png" alt="Click to see menu">
+            </div>
+            <div id="logo" >
+                <img id="logoImage" src="https://www.hubogi.com/wp-content/uploads/2017/09/Hubogi-Youtube-Yeni-Logo-ve-Sayfa-D%C3%BCzeni-Degisikliklerini-Aclikadi.jpg" alt="Logo of Youtube" >
+            </div>
+
+            <div id="textBox" >
+                <input id="inputText" type="text" size="100"  placeholder="Search">
+            </div> 
+
+            <div id="search" >
+                <button id="searchButton">
+                    <img id="searchIcon" src="images/searchIcon.png" alt="Click to search" >
+                </button>
+                
+            </div>
+            <div id="upLoad">
+                <img src="images/upLoad.png" alt="upLoad Icon" >
+            </div>
+            <div id="signIn">
+                <img src="images/signIn.png" alt="signIn Icon">
+            </div>
+
+            <!--
+                Next part is can be actived if new version of Youtube is wanted to use
+            -->
+
+            <!-- 
+            <div id="camera">
```

Comment:
- [ ] Please remove commented out code form your files, if you still want to reuse it in the future you can save it in another branch